flips/yyyymmdd-flip-template.md

```diff
@@ -0,0 +1,78 @@
+# Title of FLIP
+
+| Status        | (Proposed / Accepted / Implemented / Obsolete)       |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #**    | [NNN](https://github.com/onflow/flow/pull/NNN) (update when you have PR #)|
+| **Author(s)** | My Name (me@example.org), AN Other (you@example.org) |
+| **Sponsor**   | A N Expert (core-contributor@example.org)            |
```

Comment:
What is `A N`?

Reply:
I think it's just a placeholder name. I'll change to `AN Expert` to be consistent with `AN Other` above.

Reply:
Maybe the name of "An Expert", as initials? like "Jon Doe" = "J D" ?